README.md

```diff
@@ -1,8 +1,47 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Hive, Spark, Hadoop, C++, Python, Hadoop/HDFS
+
+**Familiar:** MySQL, Scala, NoSQL, VBA, Unix
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara/tree/master/linux_sql)]: Developed Cluster Monitoring agent which is used to monitor and collect data of various host machines connected through a network. The data that is collected includes hardware specifications, activity, and performances of machines in the cluster. Data is stored in PostgreSQL Database which is an RDBMS database provisioned with Docker Containers. Bash Scripts are used to initialize the process by creating psql container using Docker. Bash scripts are also used to monitor the host usage and host information data. This system uses crontab to collect real-time data every minute to generate statistics regarding resource utilization. The Monitoring agent sends the data to a database for storage and further analysis.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara/tree/master/core_java)]:
+      
+  - JDBC App:  Constructed the JDBC application to perform CRUD operations using DAO and DTO design patterns on a PostgreSQL retail store's database
+  - Grep App:  Programmed with Java 8 and leveraged its latest features including the stream API and lambda functions to develop an application similar to the Linux built-in grep feature. The program searches recursively through a root directory to map out all files and finds the lines that match the user-specified Java regular expression. Compilation was handled with the Maven tool, and the final product was packaged into a Docker container using Dockerfiles.
+  - The projects were designed, implemented, tested, and deployed with; *Git, Docker, Bash Scripts, CentOS 7, PostgreSQL, Java 8 [Lambda/Steam Functions], Apache Maven, Mockito, DAO Design Pattern, Spring, JDBC API, Twitter Rest API, and Google Cloud Platform*
+
+
+## Highlighted Projects
+**DataBase Design for Retail**: Designed a Relational Database and created Entity Relationship Diagram for Retail Customer containing various Entities. Implemented by creating many tables having primary keys and foreign keys to recognize the relationship Which had helped to ease the CRUD operations on the database. Performed various SQL Queries on the Database.This project was developed using IBM Db2
```

Comment:
**DataBase Design for Retail**: Designed a Relational Database and created Entity Relationship Diagram for Retail Customer containing various Entities. Implemented by creating many tables having primary keys and foreign keys to recognize the relationship, which helped ease the CRUD operations on the Database. Performed various SQL Queries on the Database.This project was developed using IBM Db2